src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -67,28 +77,35 @@ internal void InitializeLuceneConfig(string dirName, LuceneStorage storageType =
             {
                 indexDir = FSDirectory.Open(indexPath);
             }
-                
+
+
+            // Create an analyzer to process the text
+            Analyzer = CreateAnalyzerByLanguage(dynamoModel.PreferenceSettings.Locale);
```

Comment:
Added back the lucene search support for different languages.

Reply:
@reddyashish you miss to comment/remove the line 86 otherwise we will be using always the StandardAnalyzer.
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/cf650fcb-d404-4ab1-8aae-18f608b2b5db)

Reply:
Addressing it in here: https://github.com/DynamoDS/Dynamo/pull/14248